src/webview_provider/newLineagePanel.ts

```diff
@@ -365,18 +367,19 @@ export class NewLineagePanel implements LineagePanelView {
     try {
       await Promise.all([
         ...Object.values(visibleTables).map(async (node) => {
-          let compiledSql: string | undefined;
-          if (node.config.materialized === "ephemeral") {
+          const nodeType = node.uniqueId.split(".")?.[0];
+          if (
+            !(nodeType === "model" || nodeType === "snapshot") ||
+            (nodeType === "model" && node.config.materialized === "ephemeral")
+          ) {
             // ephemeral nodes can be skipped. they dont have a schema
             // and their sql makes it into the compiled sql of the models
             // referring to it.
             return;
           }
-          if (node.config.materialized !== "seed") {
-            compiledSql = await project.compileNode(node.name);
-            if (!compiledSql) {
-              return;
-            }
+          const compiledSql = await project.compileNode(node.name);
+          if (!compiledSql) {
+            return;
```

Comment:
skip this check for seed node. compiledSql will be empty for seed but it will skip the schema get below